packages/shared/components/molecules/SelectorInput.svelte

```diff
@@ -2,7 +2,7 @@
     import { localize } from '@core/i18n'
     import { InputContainer, Modal, TextInput } from 'shared/components'
 
-    export let value: string
+    export let value: string = undefined
```

Comment:
Does it make more sense to use `undefined` here rather than an empty string `''`? Both are considered falsy